.eslintrc.cjs

```diff
@@ -0,0 +1,14 @@
+module.exports = {
```

Comment:
i have this error, does anyone also have it?

<img width="727" alt="image" src="https://user-images.githubusercontent.com/31101125/203374748-df792f67-b061-4362-88da-e03dc16b5111.png">